tarantool/response.py

```diff
@@ -248,3 +251,34 @@ def __str__(self):
         return ''.join(output)
 
     __repr__ = __str__
+
+
+class ResponseExecute(Response):
+    @property
+    def lastrowid(self):
+        if self.body is None:
+            raise InterfaceError("Trying to access data, when there's no data")
+        info = self.body.get(IPROTO_SQL_INFO)
+
+        if info is None:
+            return None
+
+        lastrowids = info.get(IPROTO_SQL_INFO_AUTOINCREMENT_IDS)
+
+        return lastrowids[-1] if lastrowids else None
+
+    @property
+    def rowcount(self):
+        if self._body is None:
+            raise InterfaceError("Trying to access data, when there's no data")
+
+        info = self._body.get(IPROTO_SQL_INFO)
+
+        if info is None:
+            return -1
+
+        return info.get(IPROTO_SQL_INFO_ROW_COUNT, -1)
+
+    @property
+    def rows(self):
+        return self._data if self._body.get(IPROTO_DATA) is not None else None
```

Comment:
I guess it will fail in case of dml request where IPROTO_DATA is not provided in Tarantool response.
I propose to return just return self._body.get(IPROTO_DATA)

I think it is enough to do:
```
if self._return_code != 0:
    return None
return self._body.get(IPROTO_DATA)
```